app/helpers/github.rb

```diff
@@ -6,17 +6,36 @@ def github_create_webhook(submission)
     client = Octokit::Client.new access_token: current_user.github_access_token
     hooks = client.hooks(repo_full_name)
 
-    if !hooks || hooks.empty? || !hooks.map{|x| x.events}.any?{|e| e.include? "pull_request"}
```

Comment:
I think it was about a certain type of event that we wanted to hook on, so we would be notified when there're new comments. [PR hook event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)